src/views/BrowseView/BrowseView.style.ts

```diff
@@ -30,12 +35,26 @@ export const StyledInfiniteVideoGrid = styled(InfiniteVideoGrid)`
 export const Container = styled.div`
   position: relative;
   padding-top: ${sizes(14)};
+  overflow-x: hidden;
 `
 export const IntersectionTarget = styled.div`
   min-height: 1px;
 `
+
 export const StyledBackgroundPattern = styled(BackgroundPattern)`
   position: absolute;
-  right: 0;
+  top: 73px;
+  right: 80px;
```

Comment:
Will this value ever be used? Below `small` we're not showing anything and later it's get set to `PATTERN_OFFSET.SMALL` I think?